srml/contract/src/lib.rs

```diff
@@ -342,11 +386,14 @@ decl_storage! {
 		pub PristineCode: map CodeHash<T> => Option<Vec<u8>>;
 		/// A mapping between an original code hash and instrumented wasm code, ready for the execution.
 		pub CodeStorage: map CodeHash<T> => Option<wasm::PrefabWasmModule>;
+		/// The subtrie counter
+		pub AccountCounter: u64 = 0;
+		/// The code associated with a given account.
+		pub AccountInfoOf: map T::AccountId => Option<AccountInfo>;
 	}
 }
 
-/// The storage items associated with an account/key.
-///
+/// The storage items associated with a prefix space in a tree
```

Comment:
We need to remove `StorageOf`, as discussed [here](https://github.com/paritytech/substrate/pull/2002#discussion_r266944148) since it is not used.